features/checkout/prevent_skipping_checkout_steps.feature

```diff
@@ -14,7 +14,7 @@ Feature: Prevent skipping checkout steps
         And the store allows paying offline
         And I am a logged in customer
 
-    @ui @api
+    @ui @no-api
```

Comment:
Maybe we should consider adding this tag also for other scenarios from this file?